packages/app-project/src/screens/ProjectAboutPage/components/TeamMember/TeamMember.spec.js

```diff
@@ -21,7 +21,7 @@ describe('Component > TeamMember', function () {
     const link = getByRole('link')
     expect(link).exists()
     expect(link.href).include(
-      '/projects/zooniverse/snapshot-serengeti/users/mock_user'
+      '/zooniverse/snapshot-serengeti/users/mock_user'
```

Comment:
Wouldn't this link href still include the projects path or are these relative links?

Reply:
I think these are using the `next/link` component, which dynamically decides whether or not to use a client-side link and prefixes it with `basePath` for you. I'll double-check, but I'm pretty sure this change fixes the links on the Team page.

Reply:
Double checked, now that deploys are working again, and this change is needed to fix links to team members' profiles on the Team page.